cms/templates/index.html

```diff
@@ -449,7 +449,6 @@ <h3 class="title">${_("Were you expecting to see a particular library here?")}</
               </div>
           </div>
         </div>
-        %if course_creator_status == "granted":
```

Comment:
As with my above comment, instead of removing this gate altogether, I would create a new operation to determine if this UI control should be displayed to the user, and encapsulate any rules (such as 'libraries_enabled = True') within.